src/audio/base_fw.c

```diff
@@ -455,16 +461,86 @@ static int basefw_set_fw_config(bool first_block,
 	return 0;
 }
 
+int schedulers_info_get(uint32_t *data_off_size,
+			char *data,
+			uint32_t core_id)
+{
+	/* TODO
+	 * Core id parameter is not yet used. For now we only get scheduler info from current core
+	 * Other cores info can be added by implementing idc request for this data.
+	 * Do this if Schedulers info get ipc has uses for accurate info per core
+	 */
```

Comment:
I left this one in a bit of an awkward state. IPC works now, but it gets data only from the core that it on is currently, which is random. It was enough for my needs, though.

Reply:
but we actually need all the cores, don't we. Need a plan how to get that